wp-content/themes/core/components/video/css/video.pcss

```diff
@@ -4,11 +4,15 @@
  *
  * ----------------------------------------------------------------------------- */
 
+:root {
+	--color-video-trigger-overlay: linear-gradient(360deg, #222 0%, rgba(34, 34, 34, 0.1) 54.52%, rgba(34, 34, 34, 0) 100%);
+}
```

Comment:
assuming this remains to be done and will be handled as part of your work today, but we should have a variables partial for this component : )

Reply:
Yeppers. :100: